grand_challenge_forge/partials/example-evaluation-method/evaluate.py.j2

```diff
@@ -237,6 +335,36 @@ def write_metrics(*, metrics):
     with open(OUTPUT_DIRECTORY / "metrics.json", "w") as f:
         f.write(json.dumps(metrics, indent=4))
 
+{%- if phase.evaluation_additional_outputs | has_file %}
+def write_file(*, location, content):
+    # Write the content to a file
+    with open(location, 'w') as f:
+        return f.write(content)
+{%- endif %}
+
+{%- if phase.evaluation_additional_outputs  | has_image %}
+def write_array_as_image_file(*, location, array):
+    location.mkdir(parents=True, exist_ok=True)
+
+    # You may need to change the suffix to .tif to match the expected output
+    suffix = ".mha"
+
+    image = SimpleITK.GetImageFromArray(array)
+    SimpleITK.WriteImage(
+        image,
+        location / f"output{suffix}",
+        useCompression=True,
+    )
+{%- endif %}
+
+
+{%- if phase.evaluation_additional_outputs | has_json %}
+def write_json_file(*, location, content):
+    # Writes a json file
+    with open(location, 'w') as f:
+        f.write(json.dumps(content, indent=4))
+{%- endif %}
```

Comment:
Wouldn't it make sense to use this also for writing the metrics?

Reply:
This would also apply to the reading of predictions.json.

Having a dedicated function for this makes sense I think: they are of a different importance than the per-job values.

However, I do agree that we should re-use the json read/write functions: that is already the case for the `inputs.json`.